example/README.md

```diff
@@ -1,10 +1,10 @@
 # Demo site for React Showtime
 
-Uses `react-showtime` from `..`, so any changes there are automatically reflected here during development.
-
 Created with [Create React App](https://github.com/facebook/create-react-app).
 
 ## Development
```

Comment:
There are also development instructions in the root `README`. At first glance, they seem to be a superset of these. It seems like it might be better to have them defined in one place, or link to one from the other to minimize confusion.

Reply:
I've condensed these instructions into the root README.